lib/ClangImporter/ImportName.h

```diff
@@ -250,7 +250,8 @@ class NameImporter {
 
   /// Determine the Swift name for a Clang decl
   ImportedName importName(const clang::NamedDecl *decl,
-                          ImportNameVersion version);
+                          ImportNameVersion version,
+              clang::DeclarationName PreferredName = clang::DeclarationName());
```

Comment:
Also, please follow the prevailing style for argument names in this file (lowerCamelCase).